tests/monotouch-test/HealthKit/CategoryTypeIdentifierTest.cs

```diff
@@ -63,12 +61,13 @@ public void EnumValues_22351 ()
 				case HKCategoryTypeIdentifier.Fainting:
 				case HKCategoryTypeIdentifier.Fatigue:
 				case HKCategoryTypeIdentifier.Fever:
+				case HKCategoryTypeIdentifier.GeneralizedBodyAche:
```

Comment:
Removing #if !WATCHOS is correct.

Changing the order of preexisting identifiers is noise...fixing now.